lomap/graphgen.py

```diff
@@ -901,15 +963,23 @@ def max_dist_mol(mol):
             for each_node in self.resultGraph:
                 
                 id_mol = self.resultGraph.node[each_node]['ID']
-                mol = AllChem.RemoveHs(self.dbase[id_mol].getMolecule())
+                #sliu 07/17 skip remove Hs by rdkit if Hs cannot be removed
```

Comment:
No changelog. Though, this seems important. What will be the result if this is skipped? Why does it happen? Are we trying to get the RDKit people to fix the problem, if it is on their end?